scripts/lib/generate-crds.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+ensure_crd_gen() {
```

Comment:
Go ahead and throw this function into a file called `scripts/lib/k8s.sh` where we will put all Kubernetes helper functions. Also, rename the function `ensure_controller_gen` since we want to have the function ONLY responsible for ensuring that the `controller-gen` binary is installed.